app/assets/stylesheets/components/_collapse.scss

```diff
@@ -0,0 +1,94 @@
+@use "sass:map";
+
+.collapsed {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+
+  &.answerbox__answer-text {
+    -webkit-line-clamp: 12;
```

Comment:
I really don't like how a `-webkit-` prefix became a "standard" ...

Reply:
Nothing I can do about every browser implementing this and not implementing `line-clamp`.